packages/flutter/test/material/chip_test.dart

```diff
@@ -212,14 +230,16 @@ void main() {
     expect(tester.getSize(find.byType(Chip)), const Size(800.0, 32.0));
   });
 
-  testWidgets('Chip supports RTL', (WidgetTester tester) async {
+  testWidgets('Chip elements are ordered horizontally for locale', (WidgetTester tester) async {
+    final UniqueKey iconKey = new UniqueKey();
     final Widget test = new Overlay(
       initialEntries: <OverlayEntry>[
         new OverlayEntry(
           builder: (BuildContext context) {
             return new Material(
               child: new Center(
```

Comment:
This Center widget probably isn't needed for a correct test and it may be introducing some extra floating point variability in the chips' locations.

Reply:
Good point.  I'll take it out.